barcode-scanner-to-expo-camera.md

```diff
@@ -0,0 +1,126 @@
+## How to migrate from `expo-barcode-scanner` to `expo-camera`.
+
+`expo-barcode-scanner` is deprecated in favor of using the built in scanner in `expo-camera`.
+
+### Set up
+
+Install `expo-camera` by running the following command:
+
+```shell
+npx expo install expo-camera
+```
+
+> If you're using the bare workflow, you'll also need to run `npx pod-install`.
```

Comment:
```suggestion

[Learn more about installing and configuring expo-camera](https://docs.expo.dev/versions/latest/sdk/camera/#installation), some additional configuration is required if you don't use [CNG](https://docs.expo.dev/workflow/continuous-native-generation/).
```

we have some additional info on installing there so maybe let's just link out to this page